preinst

```diff
@@ -36,5 +36,9 @@ if [ -f "/var/lib/irods/VERSION" ] ; then
 fi
 
 if [ -f "/var/lib/irods/VERSION.json" ] ; then
-    mv /var/lib/irods/VERSION.json /var/lib/irods/VERSION.json.previous
+    mv /var/lib/irods/VERSION.json /var/lib/irods/version.json.previous
+fi
+
+if [ -f "/var/lib/irods/version.json" ] ; then
+    mv /var/lib/irods/version.json /var/lib/irods/version.json.previous
```

Comment:
consider doing the lowercase one first?   in case they *both* fire for some reason?   hmmm, maybe we should rather make the capitalized version *also* `cp` to `VERSION.json.previous`... just in case?

Reply:
Yes. I think copying the file is fine. I'm not sure what that buys us, but I don't see any harm in doing that.

Perhaps admins will feel better seeing the `VERSION.json.previous` alongside the lowercase version.

Reply:
right - we *probably* do the right thing, but in case of not... the information is not stomped/lost.

Reply:
done